### ISSUE TITLE
Cannot access private repositories

### ISSUE THREAD

**Issue by ETG153** [eclipsesource/megit#54]
When I attempt to clone a private repository that I own, the login always fails and the "not authorized" error is raised.
2FA is enabled on my profile and the method is set as Authenticator app.

**Comment by planger** [comment]
Yeah, I'm afraid you may need to create a personal access token for those use cases.
See also: https://github.com/eclipse-egit/egit/wiki/User-Guide#github-tutorial

Alternatively (and the better approach) is to use SSH with a key agent if you can.
This then works without a personal access token and MeGit would just obtain the SSH secret from the agent and, with in turn, if it is password protected again, ask for your password.
See also: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
